content/blogposts/2022/sourcegraph-4.0.md

```diff
@@ -0,0 +1,248 @@
+---
+title: "Sourcegraph 4.0"
+publishDate: 2022-09-27T10:00-07:00
+description: Announcing Sourcegraph 4.0
+tags: [blog, release]
+slug: "release/4.0"
+published: true
+heroImage: 
+socialImage: 
+changelogItems:
+---
+
+Sourcegraph 4.0, the latest version of our code intelligence platform, is now available. Our code intelligence platform includes a dozen new features to grok code, spend more time in flow, and execute big decisive refactors across your codebase.
```

Comment:
@abschu and @iskyOS - updated the intro based on Beyang's feedback in the email and the updates to the narrative blog. LMK what you think.